### PR TITLE
Add build page

### PR DESCRIPTION
**Pull request by flynnduism**
This adds the bones for the build page:

* adds the layout for build info (i.e. a list of jobs)
* adds styles, image assets
* adds basic routing (using `/build` for now until API wired up)

--

## Example data
There's a bunch of conditional data in here to show _empty state / build status / job status / log output_.

Once the app has API data this content can be better addressed to render with the appropriate context for the user. For now, I've left this static data parked in place in the template, rather than removing it and re-adding it, as we're already close to having working API data.

## Screenshot
screen shot 2018-01-23 at 2 40 48 pm

--

Closes [#119](https://github.com/Azure/kashti/issues/119). 

This page is a child of project pages (see [#121](https://github.com/Azure/kashti/issues/121)).

### CODE REVIEW

File: src/app/build/build.component.html [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+          </h1>
+
+          <div class="build-status success" ng-class="{
+            'success': build.worker.status == 'Succeeded',
+            'failed': build.worker.status == 'Failed',
+            'active': build.worker.status == 'Pending'}" style="">
+            <div class="success" ng-show="build.worker.status == 'Succeeded'" style=""><i class="icon ion-checkmark-circled"></i> Passed</div>
+            <div class="failed ng-hide" ng-show="build.worker.status == 'Failed'"><i class="icon md-close-circle"></i> Failed</div>
+            <div class="active ng-hide" ng-show="build.worker.status == 'Pending'"><img src="assets/images/rolling.svg"> In Progress</div>
+          </div>
```
Comment (seanknox) [inline src/app/build/build.component.html]: Build status might be useful as a standalone component that we could include elsewhere, like the dashboard and here. Just thinking out loud, not needed for this PR.

Comment (flynnduism) [reply src/app/build/build.component.html]: +1

**Comment by seanknox** [comment]
Looks good but I'd still like to remove the hide/show dashboard buttons and schedule the work later, unless you think there's a good reason to keep them right now.

**Comment by seanknox** [comment]
Looks like the log show/hide was removed, but I'm suggesting to hide the timeline show/hide:

kashti

**Comment by seanknox** [comment]
yeah-tgokbeqphusge

**Pull request merged**